### PR TITLE
Add support for arm64 builds

### PR DESCRIPTION
**Pull request by jackmcbarn**
The Raspberry Pi is currently beta-testing an arm64 OS, and arm64 Chromebooks can also use this via crostini.

### CODE REVIEW

**Comment by renpytom** [comment]
This will go in soon, but not for the first 7.4 (prerelease due out this weekend).

File: tasks/sysroot.py [hunk 1]
```diff
@@ -87,9 +85,21 @@ def install(c):
...
+        deb_arch = "armhf"
+        deb_mirror = "http://archive.raspbian.org/raspbian"
+        qemu_arch = "arm"
+    elif c.arch == "aarch64":
+        deb_arch = "arm64"
+        deb_mirror = "http://deb.debian.org/debian"
+        qemu_arch = "aarch64"
+
+    c.var("deb_arch", deb_arch)
+    c.var("qemu_arch", qemu_arch)
```
Comment (jackmcbarn) [inline tasks/sysroot.py:101]: I just realized that I didn't add a line for `deb_mirror` here, but when I tested it, it worked fine. What's the purpose of these calls supposed to be then?

**Comment by mal** [comment]
Think this has been superseded by 11954e5702478c98555b81303a58d865fcef2f12 and can be closed out?

**Comment by renpytom** [comment]
Yep, this can be closed.

**Pull request closed without merging**